client/src/components/payment/payment-mixin.ts

```diff
@@ -240,25 +240,34 @@ export default class PaymentMixin extends Vue {
         const receipt = sbcPayment.receipts[0]
         await paymentModule.setPaymentReceipt(receipt)
       }
+      return true
     } catch (error) {
-      if (error instanceof PaymentApiError) {
-        await errorModule.setAppError({ id: 'payment-api-error', error: error.message } as ErrorI)
-      } else {
-        await errorModule.setAppError({ id: 'fetch-nr-payment-error', error: error.message } as ErrorI)
-      }
+      console.error('fetchNrPayment() =', error) // eslint-disable-line no-console
+      await errorModule.setAppError(
+        { id: 'fetch-nr-payment-error', error: 'Could not fetch payment' }
+      )
```

Comment:
I changed a few of there over to display a user-friendly error message. Probably in the future we should change more of these.

NB: I think we don't care whether this is an API error or some other error.